build.sbt

```
@@ -104,7 +104,7 @@ lazy val allSettings = buildSettings ++
 lazy val root = project
   .in(file("."))
   .settings(buildSettings ++ dynVerSettings)
-  .aggregate(core, kinesis, pubsub, kafka, nsq, stdout, sqs, rabbitmq)
+  .aggregate(core, kinesis, pubsub, kafka, nsq, stdout, sqs, rabbitmq, integrationTests)
```

```suggestion
  .aggregate(core, kinesis, pubsub, kafka, nsq, stdout, sqs, rabbitmq)
```
More details in other comments, but the tests should probably be part of `kinesis` module, in `kinesis/src/it/`.

---

Commented on this above.